jsonschema/tests/fuzz_validate.py

```diff
@@ -0,0 +1,39 @@
+import sys
+
+from hypothesis import given
+from hypothesis import strategies as st
+import atheris
+
+import jsonschema
+
+PRIM = st.one_of(
+    st.booleans(),
+    st.integers(min_value=-(2 ** 63), max_value=2 ** 63 - 1),
```

Comment:
```suggestion
    st.integers(),
```

The 64-bit limit is an implementation issue in `ujson`, rather than a general property of JSON objects.

Reply:
Cheers!